tests/chainermn_tests/communicator_tests/test_communicator.py

```diff
@@ -772,3 +772,16 @@ def test_alltoall_cpu(self):
     def test_alltoall_gpu(self):
         self.setup(True)
         self.check_alltoall()
+
+
+@chainer.testing.attr.gpu
+def test_deprecation():
+    with chainer.testing.assert_warns(DeprecationWarning):
+        chainermn.create_communicator('hierarchical')
+
+    with chainer.testing.assert_warns(DeprecationWarning):
+        chainermn.create_communicator('two_dimensional')
+
+    if mpi_comm.is_intra:
```

Comment:
I have gotten the same error.
How about changing to the following?
https://github.com/chainer/chainer/blob/58a2a9696d4884e0b1d8eb777c55f63cabab2a38/tests/chainermn_tests/communicator_tests/test_communicator.py#L221-L224